pkg/codegen/schema/loader.go

```diff
@@ -132,6 +179,26 @@ func (l *pluginLoader) ensurePlugin(pkg string, version *semver.Version) error {
 	return nil
 }
 
+func (l *pluginLoader) Close() error {
+	for _, m := range l.mmaps {
+		if err := m.Unmap(); err != nil {
+			return err
```

Comment:
We might want to collect these errors rather than returning early s.t. we unmap/close as many files as possible.

Reply:
Good point.